DBB Cookbook/Business App Composition Recipes/BusinessApp_Device_Impacts.sql

```diff
@@ -0,0 +1,37 @@
+/*
+ * Business App Device Impacts
+ * Use this query if you are looking to discover all business apps that are impacted by a specific set of infrastructure devices and the relationships between them.
+ * You can use object_name to look at the source of the impact, or impacted_object to look at the device directly supporting the business application.
```

Comment:
Remove comma, second clause is not a complete sentence.

Reply:
okay. I'm not sure I like `source of the impact` on a second read. Any ideas on how to improve that wording?